src/main/webapp/style.css

```diff
@@ -50,13 +50,25 @@
   display: inline-block;
   z-index: 5;
   top: 60px;
-  left: 210px;
 }
 
 #videos {
   position: absolute;
   display: inline-block;
   z-index: 5;
   top: 60px;
-  left: 336px;
```

Comment:
this was the change I made to so that buttons aren't in the where the stats overlay is. right now it seems like the buttons would just sit on top of one another (since they both have top: 60px). Add these deletes back?

Reply:
they're next to each other, ill add a pic